### ISSUE TITLE
Unexpected aspect ratio format

### ISSUE THREAD

**Issue by tremby** [oscarotero/Embed#530]
Currently, you have aspect ratio is set to the height as a percentage of the width.

https://github.com/oscarotero/Embed/blob/8ac21505d048e8796c6cb9172ec5e81e5d0e0408/src/EmbedCode.php#L23

This isn't what most people would call aspect ratio; it should be width divided by height. (And not as a percentage.)

In fact, in the docs you actually say it's width divided by height:

https://github.com/oscarotero/Embed/blob/8ac21505d048e8796c6cb9172ec5e81e5d0e0408/README.md?plain=1#L61

(I noticed the unexpected value downstream here https://github.com/spicywebau/craft-embedded-assets/issues/247 where I point out it'd be a breaking change, but something to consider for a major release.)

**Comment by oscarotero** [comment]
The `ratio` property was added before CSS supported `aspect-ratio` property. The purpose was to replicate it using the padding hack (more info here: https://nikitahl.com/css-aspect-ratio)

Now that CSS supports `aspect-ratio`, probably this property doesn't make sense because you can do simply:

```php
$css_aspect_ratio = "$code->width / $code->height";
```

**Comment by tremby** [comment]
Regardless of its original purpose, and of the capabilities of CSS, and of whether you agree that most developers would be surprised to receive the current value, the logic used and the value returned do not agree with what is promised in the documentation.

**Comment by oscarotero** [comment]
Fair enough. I have edited the README.md file to improve the explanation for this property.
